### DISCUSSION TITLE
repeater usage

### DISCUSSION THREAD

**Discussion by fatihdurmaz** [filamentphp/filament#6449] [General]
return $form
            ->schema([
                Card::make()->schema([
                    Repeater::make('members')
                        ->schema([
                            Select::make('ogun_id')
                                ->relationship('ogun', 'ad')->required()->label('Öğün'),
                            Textarea::make('ad')->required()->label('Menü İçeriği'),
                            DatePicker::make('tarih')->required(),
                        ])->createItemButtonLabel('Yeni Menü Ekle')->grid(4)->cloneable()
                ])
            ]);

hello, I want to add multiple records to database with repeater. 

The error I get is SQLSTATE[HY000]: General error: 1364 Field 'ogun_id' doesn't have a default value

**Comment by danharrin** [comment]
You need to use `->relationship()`, check the repeater docs

**Comment by fatihdurmaz** [comment]
I try this method but not working8 May 2023 12:48 tarihinde Dan Harrin ***@***.***> yazdı:
You need to use ->relationship(), check the repeater docs

**Reply by emincer** [reply]
I'm having the same issue:
<img width="803" alt="Screenshot 2023-05-28 at 4 49 05 PM" src="https://github.com/filamentphp/filament/assets/1903132/bc480f1c-8468-4370-bd18-ee7ae4f46821">

and I get 
`SQLSTATE[HY000]: General error: 1364 Field 'template_id' doesn't have a default value`

@fatihdurmaz did you ever figure it out?
